### PR TITLE
Default to (localdb)\MSSQLLocalDB

### PR DESCRIPTION
**Pull request by bricelam**
Part of #189

### CODE REVIEW

File: src/EntityFramework/Infrastructure/LocalDbConnectionFactory.cs [hunk 1]
```diff
@@ -25,6 +25,14 @@ public sealed class LocalDbConnectionFactory : IDbConnectionFactory
         private readonly string _baseConnectionString;
         private readonly string _localDbVersion;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LocalDbConnectionFactory"/> class.
+        /// </summary>
+        public LocalDbConnectionFactory()
+            : this("mssqllocaldb")
```
Comment (ajcvickers) [inline src/EntityFramework/Infrastructure/LocalDbConnectionFactory.cs]: Better hope they don't change it again!

**Comment by ajcvickers** [comment]
@divega @bricelam Do you think we should announce this as a breaking change?

File: src/EntityFramework/Infrastructure/LocalDbConnectionFactory.cs (hunk 1 above)
Comment (ErikEJ) [reply src/EntityFramework/Infrastructure/LocalDbConnectionFactory.cs]: It's been the same for 4 major SQL Server versions now

**Comment by divega** [comment]
@ajcvickers Maybe we should start putting together a What's new page for EF 6.3 in the docs and put it there. But I am fine with announcing it as a breaking changes via other means as well.

**Comment by ajcvickers** [comment]
Yes, the What's New page sounds good. Let's start with that.

**Pull request merged**